collaborative_filtering/create_all_similarities.py

```diff
@@ -18,7 +19,7 @@
 global_elapsed_time = timeit.default_timer() - global_start_time
 print("The program has taken " + str(global_elapsed_time) + " seconds to load the data.")
 
-algorithms = ["cosine", "adjusted_cosine", "pearson"]
+algorithms = [similarity.COSINE, similarity.ADJUSTED_COSINE, similarity.PEARSON]
```

Comment:
This change feels like a more OOP approach but I like it.

Reply:
We should keep in mind that there are no real constants in Python even if the variables are in capital letters.